op-deployer/pkg/deployer/state/intent.go

```diff
@@ -135,6 +135,25 @@ func (c *Intent) checkL2Prod() error {
 	return err
 }
 
+type SuperchainRegistryConfig struct {
```

Comment:
These shouldn't exist in state - they should be passed in as CLI flags to the `inspect` command.

Reply:
Previously we discussed how to populate these fields. You mentioned the following, which is what I was basing this code on. But happy to make them cli flags if thats what we feel is the better ux
```
i think you should output blank values for them and tell the user to populate them in a text editor

i think it's harder to have them specify those values as flags
```

Reply:
Are you thinking we have a separate json file that they populate in a text editor, then we pass that filepath in as a cli flag?